### PR TITLE
Update fr.json

### PR DESCRIPTION
**Pull request by Orta15a**
A few new translations in fr, to avoid red parts in the sheets. Need to be approved by main translator for consistency.

### CODE REVIEW

**Comment by Orta15a** [comment]
@Alzam01 you may want to double check this.

**Comment by mlenser** [comment]
I'd recommend "Maximum réduit" to be "Max réduit" otherwise it probably won't fit.

**Comment by Alzam01** [comment]
Sorry, I didn't saw that. Will check today or tomorrow !

File: 5eShaped/translations/fr.json [hunk 1]
```diff
@@ -529,10 +529,13 @@
   "PROFICIENCIES":"Maîtrises",
   "PROFICIENCY_BONUS":"Bonus de maîtrise",
   "PROFICIENT":"Maîtrise",
+  "PSIONICS":"psioniques",
   "PSYCHIC":"psychique",
   "PUBLIC_FOR_ALL":"Public",
   "QTY":"Qté",
   "QUERY":"Varie",
+  "QUICK":"Rapide",
```
Comment (Alzam01) [inline 5eShaped/translations/fr.json:595]: Will not work 
Probably use "Visible" until something better is found

File: 5eShaped/translations/fr.json [hunk 2]
```diff
@@ -543,6 +546,8 @@
   "REACTION":"Réaction",
   "REACTIONS":"Réactions",
   "RECHARGE":"Recharge",
+  "REDUCED_MAX":"Maximum réduit",
```
Comment (Alzam01) [inline 5eShaped/translations/fr.json]: Need to be smaller : "Max réduit" or "PV max réduits". I don't like how it sound, but don't have a better idea to keep it small.

File: 5eShaped/translations/fr.json [hunk 3]
```diff
@@ -543,6 +546,8 @@
...
+  "REDUCED_MAX_HIT_POINTS":"Points de vie réduits Max",
```
Comment (Alzam01) [inline 5eShaped/translations/fr.json]: "Réduction des PV max" sounds better I think

**Review by Alzam01 — COMMENTED** [review]
Thanks for the pull request ! 
I plan to do a global check of the french translation when I find time for it, either next week or after mid-December. But if you see other errors, feel free to PM me or do more pull request, I will try to check more often for those :)

**Comment by Orta15a** [comment]
Made quite a few other addition to new vocabulary used in the sheet.
@Alzam01  please take a look and correct if needed. I had a few blanks, so I let some in English.

File: 5eShaped/translations/fr.json [hunk 4]
```diff
@@ -331,15 +341,15 @@
...
-  "CONCENTRATION_UP_TO_8_HOURS":"up to 8 hours",
+  "CONCENTRATION_UP_TO_10_MINUTES":"Concentration, jusqu'à 10 minutes",
+  "CONCENTRATION_UP_TO_12_HOURS":"Concentration, jusqu'à 12 heures",
+  "CONCENTRATION_UP_TO_1_HOUR":"Concentration, jusqu'à 1 heure",
+  "CONCENTRATION_UP_TO_1_MINUTE":"Concentration, jusqu'à 1 minute",
+  "CONCENTRATION_UP_TO_1_ROUND":"Concentration, jusqu'à 1 round",
+  "CONCENTRATION_UP_TO_24_HOURS":"Concentration, jusqu'à 24 heures",
+  "CONCENTRATION_UP_TO_2_HOURS":"Concentration, jusqu'à 2 heures",
+  "CONCENTRATION_UP_TO_6_ROUNDS":"Concentration, jusqu'à 6 rounds",
+  "CONCENTRATION_UP_TO_8_HOURS":"Concentration, jusqu'à 8 heures",
```
Comment (Orta15a) [inline 5eShaped/translations/fr.json:352]: Changed to be in sync with the english version

File: 5eShaped/translations/fr.json [hunk 5]
```diff
@@ -483,15 +522,21 @@
   "LEVELS":"niveaux",
   "LIGHTNING":"foudre",
   "LIGHT_ARMOR":"Armure Légère",
+  "LIMIT":"Limite",
   "LONG_REST":"Rep. Long",
   "MACROS":"Macros",
+  "MANIFESTING_TIME":"Manifesting time",
```
Comment (Orta15a) [inline 5eShaped/translations/fr.json:529]: No idea there...

File: 5eShaped/translations/fr.json (hunk 2 above)
Comment (Orta15a) [reply 5eShaped/translations/fr.json]: I like "Max réduit" here; it is obvious that it concerns HP, for the heart shape of the button.

File: 5eShaped/translations/fr.json (hunk 3 above)
Comment (Orta15a) [reply 5eShaped/translations/fr.json]: Agreed

File: 5eShaped/translations/fr.json [hunk 6]
```diff
@@ -353,6 +363,7 @@
   "CRIT_RANGE":"Coup Crit",
   "CURRENT":"Actuel",
   "CUSTOM":"Perso.",
+  "CUSTOM_CLASS":"Classe Custom",
```
Comment (Alzam01) [inline 5eShaped/translations/fr.json]: Change "CUSTOM" to "Custom" or "CUSTOM_CLASS" to "Classe Perso." to have the same thing

File: 5eShaped/translations/fr.json [hunk 7]
```diff
@@ -373,9 +384,33 @@
   "DICE":"Dés",
   "DIE":"Dé",
   "DISADVANTAGE":"Désavantage",
+  "DISPLAY":"Display",
```
Comment (Alzam01) [inline 5eShaped/translations/fr.json]: Affichage

File: 5eShaped/translations/fr.json [hunk 8]
```diff
@@ -373,9 +384,33 @@
...
+  "DISPLAY_NONE":"None",
```
Comment (Alzam01) [inline 5eShaped/translations/fr.json]: Aucun

File: 5eShaped/translations/fr.json [hunk 9]
```diff
@@ -515,24 +561,39 @@
...
   "PROF":"Maît",
   "PROFICIENCIES":"Maîtrises",
   "PROFICIENCY_BONUS":"Bonus de maîtrise",
   "PROFICIENT":"Maîtrise",
-  "PSYCHIC":"psychique",
+  "PSI_LIMIT":"Limite Psi",
+  "PSI_POINTS":"Points Psi",
+  "PSION":"Psion",
+  "PSIONICS":"Psioniques",
+  "PSIONICS_FULL":"Psionics Full (Psion)",
```
Comment (Alzam01) [inline 5eShaped/translations/fr.json]: Psionic Complet / Demi / Tier

**Comment by mlenser** [comment]
Is this ready to be merged? I ignored it while there were several changes

**Comment by Orta15a** [comment]
I made all the changes suggested by @Alzam01 so I think it's good to merge !

**Pull request merged**